js/layout-javascript/simple-list-code.js

```diff
@@ -2428,6 +2435,11 @@ DynamicList.prototype.showDetails = function(id, listData) {
     });
   }
 
+  $(_this.$detailsContent).mousedown(function(event) {
```

Comment:
This doesn't take into account that a user may use TAB to bring their focus to a link within the overlay, then press ENTER. That will also end up scrolling the user to the top.